src/ocm/components/AddHosts/utils.ts

```diff
@@ -7,14 +8,18 @@ export const canAddHost = ({ cluster }: { cluster: OcmClusterType }) => {
     if (cluster.aiCluster) {
       if (!isSNO(cluster.aiCluster)) {
         return cluster.aiCluster.status === 'installed';
+      } else {
```

Comment:
It would be more readable to invert the conditions to avoid the double negation on the "else"
```suggestion
      if (isSNO(cluster.aiCluster)) {
       return (
          cluster.aiCluster.status === 'installed' &&
          canAddHostSNO(cluster.aiCluster.openshiftVersion || cluster.openshift_version)
        );
      } else {
          return cluster.aiCluster.status === 'installed';
      }
```